tools/KoreBuild.Console/Commands/DockerFiles/jessie.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/dotnet:2.0-runtime-deps-jessie
+
+RUN apt-get update && \
+    apt-get install -y git && \
+    apt-get install -y curl && \
+    apt-get install -y unzip && \
+    apt-get install -y apt-transport-https
```

Comment:
Add `rm -rf /var/lib/apt/lists/*` to the end of this chain. See https://github.com/anurse/ConferencePlanner/blob/735d5ef3352d18c7a258267a1f18aad359479ec2/docker/base.Dockerfile#L9

That deletes the files `apt-get update` downloaded, which are not insignificant. You need to chain it in to the `RUN` command so that it all ends up being baked as one layer in the image. That ensures that the layer includes only the files added by the packages you installed, and no other cruft.

Reply:
You also only need one call to `apt-get install`.  Example:

https://github.com/aspnet/benchmarks/blob/dev/docker/benchmarks/Dockerfile#L3

Reply:
👍 more a performance that space saving issue, but definitely a good one.